src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -1,6 +1,8 @@
 package edu.kis.vh.nursery.stack;
 
-public class IntArrayStack implements IntArrayStackInterface {
+public class IntArrayStack implements IntStackInterface {
+    private final int SIZE = 12;
+    private final int REAL_SIZE = SIZE + 1;
```

Comment:
You misunderstood the comment. 
Constant REAL_SIZE should be moved to IntStackInterface. 
A better name would be REAL_SIZE->CAPACITY and SIZE->FULL_INDEX